c/meterpreter/source/extensions/priv/namedpipe_efs.c

```diff
@@ -137,8 +137,14 @@ DWORD WINAPI trigger_efs_connection(LPWSTR pPipeName)
 		if (hr == ERROR_BAD_NETPATH) {
 			dprintf("[ELEVATE] trigger_efs_connection: Success");
 		} else {
-			dwResult = hr;
-			dprintf("[ELEVATE] trigger_efs_connection: Did not receive expected output. Attack might have failed.");
```

Comment:
The previous `if` path for success doesn't look to be breaking out of the loop either on success which is a little concerning.